helm-deployment/ldap/README.md

```diff
@@ -0,0 +1,7 @@
+## IMM Test ldap
+
+In this directory are stored files directly needed to launch Openldap helm chart used by our functional and e2e tests.
```

Comment:
i am not sure if we want to explicitly say, that this Ldap is only for test. It is also useful for demo purposes.